src/bundle/Resources/public/js/scripts/helpers/tooltips.helper.js

```diff
@@ -15,7 +15,9 @@
                 };
                 const extraClasses = tooltipNode.dataset.extraClasses || '';
                 const placement = tooltipNode.dataset.placement || 'bottom';
-                const container = tooltipNode.dataset.tooltipContainerSelector || 'body';
+                const container = tooltipNode.dataset.tooltipContainerSelector ?
+                    $(tooltipNode).closest(tooltipNode.dataset.tooltipContainerSelector) :
```

Comment:
```suggestion
                    tooltipNode.closest(tooltipNode.dataset.tooltipContainerSelector) :
```

You don't have to use jQuery for closest.